### DISCUSSION TITLE
使用mysqldump数据导出表结构LOCK TABLES错误

### DISCUSSION THREAD

**Discussion by kkkeekkkee** [apache/doris#9048] [A - General / Q&A]
![微信图片_20220415170728](https://user-images.githubusercontent.com/38931421/163551536-e4b4874e-d39e-4cf3-a2a3-fb257ba92f9a.png)

![image](https://user-images.githubusercontent.com/38931421/163551733-56eec988-b4e8-4056-8c56-81fe86c807a5.png)

**Comment by hf200012** [comment]
This is only supported in version 1.0
